src/cocotb/_write_scheduler.py

```diff
@@ -62,6 +62,10 @@ def schedule_write(
         write_func: Callable[..., None],
         args: Sequence[Any],
     ) -> None:
+        if cocotb.sim_phase == cocotb.SimPhase.READ_ONLY:
+            raise RuntimeError(
+                f"Write to object {handle._name} was scheduled during a read-only sync phase."
```

Comment:
```suggestion
                f"Write to object {handle._name} was scheduled during a read-only simulation phase."
```
Was this initially a "sim"/"sync" mixup?